index.js

```diff
@@ -0,0 +1,2 @@
+require('./format-as-currency');
```

Comment:
this doesn't seem like correct behavior. without an export, i'm not sure what's getting imported. i'll play around with format-as-currency + webpack tonight, and make sure this change works for browserify and other bundlers too.